src/general/OptionsMenu.tscn

```diff
@@ -645,11 +646,21 @@ custom_styles/separator = SubResource( 6 )
 margin_left = 245.0
 margin_right = 549.0
 margin_bottom = 35.0
-disabled = true
 text = "Reset ALL to Default Settings"
 __meta__ = {
 "_edit_use_anchors_": false
 }
+
+[node name="ResetConfirm" type="ConfirmationDialog" parent="CenterContainer"]
+visible = true
+margin_left = 340.0
+margin_top = 324.0
+margin_right = 940.0
+margin_bottom = 395.0
+rect_min_size = Vector2( 600, 70 )
+popup_exclusive = true
+window_title = "Are you sure you want to reset ALL settings to default values?"
+dialog_text = "Are you sure?"
```

Comment:
The dialog text should be the longer one "Are you sure you want to reset ALL settings to default values?"
and the title should be "Reset settings to defaults?"